### ISSUE TITLE
A few accessibility issues

### ISSUE THREAD

**Issue by marcysutton** [yarnpkg/website#197]
Since @thejameskyle asked on Twitter, I found a few website accessibility issues using the [axe-chrome extension](https://chrome.google.com/webstore/detail/axe/lhdoppojpmngadmnindnejefpokejbdd?hl=en-US):

### Frames must have a title attribute
Not sure if you can control this, but the Github Star iframe needs a unique title. That will label it in the list of page landmarks

### HTML element must have a lang attribute
Add 'en-us' or whatever locale you're serving to indicate the language on the page. Helps screen readers with pronunciation

### Form elements must have labels
The search input is missing a label–placeholder isn't reliably exposed. You could use `aria-label` but an explicit label is best.

Depending on the website stack (which I haven't dug into yet) you could integrate [axe-core](https://github.com/dequelabs/axe-core) or  [axe-webdriverjs](https://github.com/dequelabs/axe-webdriverjs) to programmatically test for these same conditions.

---

Great job with the color contrast changes already, they look great!

**Comment by thejameskyle** [comment]
Thanks! I've fixed the problems that I can and a PR adding lang=en issue to the github-button project https://github.com/mdo/github-buttons/pull/107

**Comment by marcysutton** [comment]
The Github thing needs a title like "View on Github", the HTML element needs the lang.
